### DISCUSSION TITLE
How to use optuna to implement learning rate decay?

### DISCUSSION THREAD

**Discussion by ZhaiJojo** [optuna/optuna#4471] [Q&A] [answered]
I am new to optuna and are using optuna with pytorch. I am wondering if it is feasible to implement learning rate decay with optuna? I have searched the Internet and got no answer. And before that question, is it helpful or necessary to use learning rate decay if we have already used optuna to adjust the learning rate of the Adam optimizer? Thank you very much for any help~

**Comment by nzw0301** [comment, marked as answer]
I suppose an easy way to 

1. Select a scheduler  from https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate with `suggest_categorical`
2. Select suggested learning rate scheduler's parameters
